environment.yml

```diff
@@ -0,0 +1,37 @@
+name: lfc_analysis_env
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - blinker=1.4=py_1
+  - brotlipy=0.7.0=py39hb82d6ee_1001
+  - ca-certificates=2021.5.30=h5b45459_0
+  - certifi=2021.5.30=py39hcbf5309_0
+  - cffi=1.14.5=py39h0878f49_0
+  - chardet=4.0.0=py39hcbf5309_1
+  - cryptography=3.4.7=py39hd8d06c1_0
+  - idna=2.10=pyh9f0ad1d_0
+  - mwclient=0.10.1=pyhd8ed1ab_0
+  - oauthlib=3.1.1=pyhd8ed1ab_0
+  - openssl=1.1.1k=h8ffe710_0
+  - pip=21.1.2=pyhd8ed1ab_0
+  - pycparser=2.20=pyh9f0ad1d_2
+  - pyjwt=2.1.0=pyhd8ed1ab_0
+  - pyopenssl=20.0.1=pyhd8ed1ab_0
+  - pysocks=1.7.1=py39hcbf5309_3
+  - python=3.9.4=h7840368_0_cpython
+  - python_abi=3.9=1_cp39
+  - requests=2.25.1=pyhd3deb0d_0
+  - requests-oauthlib=1.3.0=pyh9f0ad1d_0
+  - setuptools=49.6.0=py39hcbf5309_3
+  - six=1.16.0=pyh6c4a22f_0
+  - sqlite=3.35.5=h8ffe710_0
+  - tzdata=2021a=he74cb21_0
+  - urllib3=1.26.5=pyhd8ed1ab_0
+  - vc=14.2=hb210afc_4
+  - vs2015_runtime=14.28.29325=h5e1d092_4
+  - wheel=0.36.2=pyhd3deb0d_0
+  - win_inet_pton=1.1.0=py39hcbf5309_2
+  - wincertstore=0.2=py39hcbf5309_1006
+prefix: C:\Users\isabe\Anaconda\envs\lfc_analysis_env
```

Comment:
This line is specific to your local instance and should be removed -- I don't know conda well enough to know what the purpose of this line is, so will rely on you to research it.

Reply:
I have switched to pipenv and this file will no longer be necessary.